tests/tests_suite_test.go

```diff
@@ -53,14 +53,18 @@ func TestTests(t *testing.T) {
 	flags.NormalizeFlags()
 	tests.CalculateNamespaces()
 	maxFails := getMaxFailsFromEnv()
-	artifactsPath := path.Join(flags.ArtifactsDir, "k8s-reporter")
-	junitOutput := path.Join(flags.ArtifactsDir, "junit.functest.xml")
+	artifactsPath := filepath.Join(flags.ArtifactsDir, "k8s-reporter")
+	junitOutput := filepath.Join(flags.ArtifactsDir, "junit.functest.xml")
 	if qe_reporters.JunitOutput != "" {
-		junitOutput = qe_reporters.JunitOutput
+		junitOutput = filepath.Join(flags.ArtifactsDir, qe_reporters.JunitOutput)
 	}
 	if config.GinkgoConfig.ParallelTotal > 1 {
-		artifactsPath = path.Join(artifactsPath, strconv.Itoa(config.GinkgoConfig.ParallelNode))
-		junitOutput = path.Join(flags.ArtifactsDir, fmt.Sprintf("partial.junit.functest.%d.xml", config.GinkgoConfig.ParallelNode))
+		artifactsPath = filepath.Join(artifactsPath, strconv.Itoa(config.GinkgoConfig.ParallelNode))
+		if qe_reporters.JunitOutput != "" {
```

Comment:
Would it be possible to treat both reporters the same way? If run in parallel we dump the partial xmls in the artifact directory with well known names? I don't mind too much, but want to avoid too much special casing.

Reply:
Not sure I understand completely. I am all for well known names, but isn't the point why `qe_reporters.JunitOutput` exists to make in configurable? (TBH not sure why it is there, maybe @lukas-bednar can shed some light here?)

Reply:
@rmohr could you please take another look at this? Thank you!